Gemfile

```diff
@@ -42,6 +42,7 @@ group :bench do
 end
 
 group :test do
+  gem 'rspec-rails'
```

Comment:
I'll move it to the gemspec. Didn't understand your second comment, maybe is missing a link?